rules/opentitan/cc.bzl

```diff
@@ -344,6 +353,10 @@ opentitan_test = rv_rule(
             executable = True,
             cfg = "exec",
         ),
+        "binary": attr.label(
+            allow_files = True,
+            doc = "Use a binary for this test rather than compiling srcs",
```

Comment:
This doc string makes it sound like `binary` is a bool that specifies whether or not to use a binary, rather than the binary itself. How about `The binary to use for this test, rather than compiling srcs`.

Reply:
Fixed.